main.py

```diff
@@ -119,9 +121,17 @@ def list_schedules():
     """
     schedules_list = []
     with client.context():
-        keys = SchedulesModel.query().fetch(keys_only=True)
-        for key in keys:
-            schedules_list.append(key.id())
+        verbose = request.args.get("verbose") == "true"
+        if (verbose):
+            schedules = SchedulesModel.query().fetch(keys_only=False)
```

Comment:
`keys_only` has a default of `None`. If we don't want it true, we can just omit it entirely.

Reply:
fixed, thanks.